src/gui/plugins/apply_force_torque/ApplyForceTorque.qml

```diff
@@ -88,12 +91,21 @@ GridLayout {
     }
   }
 
-  // Force
+  // Force and offset
   Text {
-    Layout.columnSpan: 8
+    Layout.columnSpan: 4
     id: forceText
     color: "dimgrey"
-    text: qsTr("Force (applied to the center of mass):")
+    text: qsTr("Force:")
+  }
+
+  Text {
+    Layout.columnSpan: 4
+    Layout.fillWidth: true
+    wrapMode: Text.WordWrap
+    id: offsetText
+    color: "dimgrey"
```

Comment:
Can we change all the `dimgrey`s to black?

Reply:
Done in 0a7e7b3